core/domain/src/main/java/com/google/samples/apps/nowinandroid/core/domain/GetUserNewsResourcesUseCase.kt

```diff
@@ -51,6 +55,56 @@ class GetUserNewsResourcesUseCase @Inject constructor(
         }.mapToUserNewsResources(userDataRepository.userData)
 }
 
+class GetFollowedUserNewsResourcesUseCase @Inject constructor(
```

Comment:
Have you considered adding an additional argument to the `GetUserNewsResourcesUseCase` instead of creating new use case? 
I we go the "separate use case" route, can we put each use case in a separate file? I think there's no reason to deviate from this convention.

Reply:
> Have you considered adding an additional argument to the `GetUserNewsResourcesUseCase` instead of creating new use case?
This wouldn't work because `GetUserNewsResourcesUseCase` (or the logic it contains) needs to be called every time the followed topics change. 
>  I we go the "separate use case" route, can we put each use case in a separate file?
Good suggestion. Done